### PR TITLE
Win x64 dependencies via scoop

### PR DESCRIPTION
**Pull request by JoeMeeks**
+ readme scoop instructions for x64
+ scoop x64 executable relative path
+ scoop x64 installation error message

### CODE REVIEW

File: src/adapters/iosAdapter.ts [hunk 1]
```diff
@@ -143,12 +143,14 @@ export class IOSAdapter extends AdapterCollection {
         debug(`iOSAdapter.getProxyPath`)
         return new Promise((resolve, reject) => {
             if (os.platform() === 'win32') {
-                const proxy = os.arch() === 'x64' ? path.resolve(__dirname, '../../node_modules/vs-libimobile/x64/ios_webkit_debug_proxy.exe') : path.resolve(__dirname, '../../node_modules/vs-libimobile/lib/ios_webkit_debug_proxy.exe');
+                const x64 = os.arch() === 'x64';
+                const proxy = x64 ? path.resolve(__dirname, '../../../../../../../scoop/apps/ios-webkit-debug-proxy/current/ios_webkit_debug_proxy.exe') : path.resolve(__dirname, '../../node_modules/vs-libimobile/lib/ios_webkit_debug_proxy.exe');
```
Comment (auchenberg) [inline src/adapters/iosAdapter.ts:147]: This path doesn't seem right. See https://github.com/lukesampson/scoop/wiki/Global-Installs

**Comment by JoeMeeks** [comment]
This path did resolve properly to my %UserProfile%\scoop\apps\ios-webkit-debug-proxy directory.

**Comment by auchenberg** [comment]
@JoeMeeks I would imagine it should more be something like https://stackoverflow.com/questions/9080085/node-js-find-home-directory-in-platform-agnostic-way/32556337#32556337  as `ios_webkit_debug_proxy.exe` should be installed globally

**Comment by auchenberg** [comment]
Closing due to inactivity.

**Pull request closed without merging**